docs/ops/config.rst

```diff
@@ -29,9 +29,24 @@ each section of the user config file.
 Runtime Config
 ==============
 
+Data Cube needs to be told which database index to connect to. This can be done via a file with profiles,
+or directly via environment variables.
+
+Environment Variables
+---------------------
+
+DATACUBE_DB_URL:
+   postgresql://user:password@host/database
+
+
+- Else look for ``DB_HOSTNAME``, ``DB_USERNAME``, ``DB_PASSWORD``, ``DB_DATABASE``
```

Comment:
Thanks for this PR @omad, nice to be able to see docs rendered as part of the PR.

Pretty happy with all of those changes. Just this section about environment is already documented at the bottom of this file. So if you can combine the two parts somehow that would be great. I guess it's a kind of "merge conflict".

Reply:
I'll fix it up. That's what happens when branches live for too many months.